src/psyclone/psyGen.py

```diff
@@ -1021,15 +1021,15 @@ def gen_code(self, parent):
         self.parent._symbol_table = self.parent.symbol_table.shallow_copy()
         # pylint: enable=protected-access
 
-        # Global symbols promoted from Kernel Globals are in the SymbolTable
-        # First aggregate all globals variables from the same module in a map
+        # Imported symbols promoted from Kernel imports are in the SymbolTable.
+        # First aggregate all imports variables from the same module in a map.
```

Comment:
"...all variables imported from the same module in a map."

Reply:
Done